src/pip/_internal/commands/search.py

```diff
@@ -118,8 +130,13 @@ def print_results(hits, name_column_width=None, terminal_width=None):
             target_width = terminal_width - name_column_width - 5
             if target_width > 10:
                 # wrap and indent summary to fit terminal
-                summary = textwrap.wrap(summary, target_width)
-                summary = ('\n' + ' ' * (name_column_width + 3)).join(summary)
+                # Since summary is of type str, and textwrap.wrap
+                # returns a List[str], temporary save the output to
+                # summary_lines, and assign back to summary
+                # once we join summary_lines to a string
```

Comment:
This comment does not add much value. The `summary_lines` name explains it by itself.

Reply:
I have removed the comment